SeaPublicWebsite/Models/EnergyEfficiency/QuestionFlowViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using SeaPublicWebsite.Helpers.UserFlow;
+
+namespace SeaPublicWebsite.Models.EnergyEfficiency
+{
+    public class QuestionFlowViewModel
```

Comment:
If we're never expecting this class to be used directly then you can make it `abstract`

Reply:
We indeed never use it. I'll make it abstract